src/minuteman_lashup_vip_listener.erl

```diff
@@ -255,13 +257,17 @@ rewrite_name({{tcp, {name, {Name, FWName}}, PortNum}, BEs}) ->
 rewrite_name(Else) ->
     Else.
 
-retry_spartan(State) ->
-    push_state_to_spartan(State).
+retry_spartan(State0) ->
+    State1 = push_state_to_spartan(State0),
```

Comment:
@nlsun I am changing the state in push_state_to_spartan which needs to be carry on.